HW1/P5/P5.txt

```diff
@@ -1 +1,46 @@
 # Your discussion here
+
+Part1 shortest paths
+
+I ran two times and got two results:
```

Comment:
Why does running it 2x give 2 different results (rather than the same result or all the results)?